tools/expotools/src/versioning/ios/transforms/kernelFilesTransforms.ts

```diff
@@ -0,0 +1,54 @@
+import { TransformPipeline } from '.';
+
+/**
+ * These modification will be run against `ios/Exponent/kernel` directory.
+ * If you need to modify other files from the `ios` directory then find
+ * a better place for it or refactor the function that depends on this list.
+ * The nature of these changes is that they're not permanent and at one point
+ * of time (SDK drop) these should be rollbacked.
+ * @param versionName e.g. 21.0.0, 37.0.0, etc.
+ * @param rollback This flag indicates whether the change should be rollbacked.
+ */
+export function kernelFilesTransforms(
+  versionName: string,
+  rollback: boolean = false
+): TransformPipeline {
+  return {
+    transforms: [
+      {
+        paths: ['EXAppViewController.m'],
+        ...withRollback(rollback, {
+          replace: /(?<=#import <React\/RCTAppearance\.h>)/,
+          with: `\n\n#if __has_include(<${versionName}React/${versionName}RCTAppearance.h>)\n#import <${versionName}React/${versionName}RCTAppearance.h>\n#endif`,
```

Comment:
Just to make it consistent with the second transform 😉 
```suggestion
          with: `\n#if __has_include(<${versionName}React/${versionName}RCTAppearance.h>)\n#import <${versionName}React/${versionName}RCTAppearance.h>\n#endif`,
```